### DISCUSSION TITLE
Observed vs. predicted or predicted vs. observed?

### DISCUSSION THREAD

**Discussion by wilbertdew** [Open-Systems-Pharmacology/Forum#1460] [Feature Requests & Ideas]
Hi everyone,

I would like to discuss the option of switching the axes in the observed vs predicted plot.
There are several reasons to do so, most of which are explained in this [article](https://www.sciencedirect.com/science/article/abs/pii/S0304380008002305).
In short, the dependent variable goes typically on the y-axis, which should be the observed.
In PK-Sim, the observed is currently placed on the x-axis. This is also not clearly indicated, which enhances the potential problems with this orientation. As an illustration, a hypothetical but not unlikely scenario is shown for a situation where data is available at only one time point for multiple individuals, and they are distributed around the predicted geometric mean value at that time point. The right-hand plot shows the currently implemented axis orientation in PK-Sim, where one could think there is a model misspecification because of an observed trend: A trendline would show a horizontal line instead of the desired diagonal around the line of identity. The left-hand plot shows a good distribution around the predicted value, and a trendline is uninformative for this dataset.
Let me know what your thoughts are on this.
Best,

Wilbert

![image](https://user-images.githubusercontent.com/8969023/221411436-98e7e48b-d05c-49e8-952c-4f5000a111d5.png)

**Comment by Yuri05** [comment]
> In PK-Sim, the observed is currently placed on the x-axis. This is also not clearly indicated, which enhances the potential problems with this orientation. 

Let me disagree with you regarding this statement. It is clearly indicated in the axis description of predicted vs. observed plots.
![grafik](https://user-images.githubusercontent.com/25061876/221431372-3ce1f81d-51e6-47af-84c3-797d7c5763d4.png)

**Comment by wilbertdew** [comment]
thanks @Yuri05, is that added in v11.1? I am using v11.0 and I dont have this, neither in PK-Sim (first figure) nor MoBi (second figure)

![image](https://user-images.githubusercontent.com/8969023/221500361-6d3bddbb-4d62-4560-a364-bd61adfd495b.png)
![image](https://user-images.githubusercontent.com/8969023/221501181-ec1032d3-3b4f-4574-9073-fc8a84d26819.png)

**Comment by Yuri05** [comment]
>is that added in v11.1

Seems so :)

**Comment by wilbertdew** [comment]
@Yuri05 Would you consider swapping the axis can/should be implemented?

**Reply by Yuri05** [reply]
Personally, I don't see any benefit - but if you think it would be valuable, please create a feature request and it will be added to the discussion list.

**Reply by wilbertdew** [reply]
Is the categorization as a feature request here sufficient, or do I need to do that under "issues"?

**Comment by PavelBal** [comment]
Created an issue here https://github.com/Open-Systems-Pharmacology/OSPSuite.Core/issues/2759
